sycl/doc/extensions/ExplicitSIMD/README.md

```diff
@@ -89,5 +91,5 @@ program behavior if violated.
 
 ##### Other restrictions:
 - Only Intel GPU device is supported
-- Usual and ESIMD DPC++ kernels can not co-exist in the same application in most
-  cases
+- Interoperability between regular SYCL and ESIMD kernels is not supported.
+  I.e., it's not possible to call ESIMD code from SYCL and vise-versa.
```

Comment:
We can in fact call SYCL functions with bodies from ESIMD, so please rephrase.

Reply:
Rephrased.